client/src/components/Cart/Stepper/StepIconStyles.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import FlightTakeoffIcon from "@material-ui/icons/FlightTakeoff";
+import LocalHotelIcon from "@material-ui/icons/LocalHotel";
+import DriveEtaIcon from "@material-ui/icons/DriveEta";
+
+import clsx from "clsx";
+
+const useColorlibStepIconStyles = makeStyles({
+  root: {
+    backgroundColor: "#ccc",
+    zIndex: 1,
+    color: "#fff",
+    width: 30,
+    height: 30,
+    display: "flex",
+    borderRadius: "50%",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  active: {
+    backgroundColor: "#ffb347",
+    boxShadow: "0 4px 10px 0 rgba(0,0,0,.25)",
+  },
+  completed: {
+    backgroundColor: "#ffb347",
+  },
+});
+
+const StepIconStyles = (props) => {
+  const classes = useColorlibStepIconStyles();
+  const { active, completed } = props;
+
+  const icons = {
+    1: <FlightTakeoffIcon />,
+    2: <LocalHotelIcon />,
+    3: <DriveEtaIcon />,
+  };
```

Comment:
Instead of using numbers here (which is not accurate anyways, because they will get coerced into strings - objects can't have numbers as keys -), I would suggest using descriptive names. Otherwise, in another file when you're using this component, and you don't have documentation available, you might forget which number represents which icon. But if you do for example:

```javascript
const icons = {
  "plane": <FlightTakeoffIcon />,
  "hotel": //...,
  "car": //...
};
```

You could then do

```javascript
<StepIconStyles icon="car" active />
```

instead of

```javascript
<StepIconStyles icon="3" active />  // 3? what is that "magic number"?
```

Reply:
Hey John, thanks for your reviews. I don't pass each individual icon prop into the Component. The function is passed in the StepIconComponent prop.

`           <StepLabel StepIconComponent={StepIconStyles}>{label}</StepLabel>
`